addons/loyalty/views/loyalty_reward_views.xml

```diff
@@ -37,14 +37,6 @@
 
                         <group string="Discount" invisible="reward_type != 'discount' or program_type in ('gift_card','ewallet')">
                             <field name="discount_max_amount"/>
-                            <field
-                                name="tax_ids"
-                                placeholder="Untaxed discount"
-                                widget="many2many_tags"
-                                options="{'no_create': True}"
-                                invisible="discount_applicability != 'order' or
-                                    discount_mode not in ('per_order','per_point')"
-                            />
```

Comment:
I don't think it's a good idea for the """stable""" policy. Maybe you should just put them invisible to be sure to not break any xpath?